### ISSUE TITLE
Email input not showing value using modelForm

### ISSUE THREAD

**Issue by bbashy** [spatie/laravel-html#60]
Hi,

Think I found a bug in the email type value.

```
{{ html()->email('email') }}

// generates
<input type="email" name="email" id="email" value>
```
```
{{ html()->text('email') }}
// or
{{ html()->input('email', 'email') }}

// generates
<input type="text" name="email" id="email" value="test@test.com">
<input type="email" name="email" id="email" value="test@test.com">
```

**Comment by bbashy** [comment]
Could be related to #59

**Comment by sebastiandedeyne** [comment]
Managed to reproduce this with a failing test, gonna take a look now

**Comment by bbashy** [comment]
Many thanks! I've temporarily fixed it by using the input method.

**Comment by sebastiandedeyne** [comment]
Tagged a fix in `2.8.2` :)
